packages/react-liveness/src/components/FaceLivenessDetector/utils/__tests__/platform.test.ts

```diff
@@ -0,0 +1,19 @@
+import { VERSION } from '../../../../version';
+import { getLivenessUserAgent } from '../platform';
+
+describe('getLivenessUserAgent', () => {
+  it('should return a user agent string containing the current version', () => {
+    const customUserAgent = getLivenessUserAgent();
+
+    expect(customUserAgent).toContain('ui-react-liveness');
+    expect(customUserAgent).toContain(VERSION);
+  });
+
+  it('should return a user agent string with custom content', () => {
+    const customUserAgent = getLivenessUserAgent('specialEnding');
+
+    expect(customUserAgent).toContain('ui-react-liveness');
+    expect(customUserAgent).toContain(VERSION);
+    expect(customUserAgent).toContain('specialEnding');
+  });
```

Comment:
Do we plan to use custom content, and if not, should we not worry about it for now?

Reply:
I'm open to removing it, I mostly kept it in to follow the same pattern as the JS team but can't think of a good reason for it either

Reply:
Yep, let's remove it then 😈